acceptance/run.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+CONFIG_FILE=config
+SAMPLE_APP_DIR=sample-android-mixpanel-integration
+TESTS=(test.py)
+NODE_SERVER_PID=NULL
+SELEDROID_PID=NULL
+NPM_PACKAGES_REQUIRED=(express)
+
+VERBOSE=0
+
+function npm_package_check {
+	for package in $NPM_PACKAGES_REQUIRED
+	do
+		ls node_modules 2> /dev/null | grep $package > /dev/null
+		if [ $? != 0 ]
+		then
+			echo "$package is missing. installing.."
+			if [ $VERBOSE != 0 ]
+			then
+				sudo npm install $package --save
```

Comment:
Let's not put sudo into these scripts- I believe we should be able to install npm packages locally.